pkg/minikube/cluster/cluster.go

```diff
@@ -206,9 +206,17 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 		driver = createVirtualboxHost(config)
 	case "vmwarefusion":
 		driver = createVMwareFusionHost(config)
-	case "kvm", "kvm2":
+	case "kvm":
```

Comment:
It may be good to allow a user to suppress these messages so we don't break any automated scripts?  I think that was an issue with some of the other warning messages we've added.  We might be able to reuse one of our env vars or config values for it.

Reply:
Done.